extension/src/service/web-component-service.js

```diff
@@ -104,7 +104,7 @@ function extendRequestObjWithApplicationInfo(requestObj) {
 
 function extendRequestObjWithMetadata(requestObj, commercetoolsProjectKey) {
   requestObj.metadata = {
-    commercetoolsProjectKey,
+    ctProjectKey: commercetoolsProjectKey,
```

Comment:
Maybe we should document here the restrictions on the length of the metadata fields?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/665/commits/f68c497458000839c6e212c4d75a15eefb749a00